apps/dashboard/src/components/workflow-editor/steps/email/email-editor.tsx

```diff
@@ -21,6 +23,8 @@ export const EmailEditor = (props: EmailEditorProps) => {
             getComponentByType({
               component: disableOutputSanitization.component,
             })}
+          {isHtmlEditorEnabled &&
+            getComponentByType({ component: editorType?.component ?? UiComponentEnum.EMAIL_EDITOR_SELECT })}
```

Comment:
It's safe only to feature flag this select, without it the users won't have an option to see the HTML editor